addons/account_edi_ubl_cii/models/res_partner.py

```diff
@@ -71,7 +71,7 @@ class ResPartner(models.Model):
             ('0201', "0201 - Codice Univoco Unità Organizzativa iPA"),
             ('0202', "0202 - Indirizzo di Posta Elettronica Certificata"),
             ('0204', "0204 - Leitweg-ID"),
-            ('0208', "0208 - Numero d'entreprise / ondernemingsnummer / Unternehmensnummer"),
+            ('0208', "0208 - Belgium Numero d'entreprise / ondernemingsnummer / Unternehmensnummer"),
```

Comment:
No?
![image](https://github.com/odoo/odoo/assets/21679159/0c6f218e-3711-4656-bcf6-226a5c0eabdf)

Reply:
The reason why we decided to add this is because even though 0208 is precomputed for users, they don't understand that it is a Belgian EAS code and look for the 9925, which is less preferred :slightly_frowning_face: we have 3 Belgian participants right now who registered with 9925

Reply:
Nevertheless, please remove this part since the warnings do the job. I prefer we merge it today than wait for one more week. Laurent can then R+ ; all the rest being ok. Banner + precomputation are great enough.